src/api/track.js

```diff
@@ -36,9 +41,12 @@ class Track {
       sdkVersion: SDK_VERSION,
       stopped: true,
       userId,
+      tripOptions,
     };
 
-    const response = await Http.request('POST', `v1/track`, body);
+    const trackEndpoint = Cookie.getCookie(Cookie.TRACK_ENDPOINT) || 'v1/track';
```

Comment:
do we need this to be customizable? I was under the impression it'd only be relevant for the host

Reply:
yeah we're going to expose a new track endpoint under /app that will use session auth for mock tracking in the dashboard (diff rate limits, etc)

Reply:
ah, good point